cmd/secret/cmd/root.go

```diff
@@ -100,31 +108,48 @@ func (r *root) setCmd() *cobra.Command {
 	var value string
 	var path string
 	var redisURL string
+	var postgresURL string
+	var migration string
 	var setCmd = &cobra.Command{
 		Use:   "set",
 		Short: "Saves data to the specified storage in encrypted form",
 		Long:  "it takes keys and a value from user and saves value in encrypted manner in specified storage",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			var ds secretApi.DataSaver
 			var cr = crypto.NewCryptographer([]byte(cipherKey))
-			rdb := redis.NewClient(&redis.Options{Addr: redisURL, Password: "", DB: 0})
-			err := rdb.Ping(r.cmd.Context()).Err()
-			if err != nil {
-				return fmt.Errorf("redis db is not reachable:  %w", err)
-			}
-
+			logger := r.logger.Named("set cmd")
```

Comment:
Better use phrases that are not separated by spaces,
```suggestion
			logger := r.logger.Named("set-cmd")
```